src/test/java/teammates/test/cases/webapi/GetTimeZonesActionTest.java

```diff
@@ -1,13 +1,16 @@
 package teammates.test.cases.webapi;
 
 import org.apache.http.HttpStatus;
+import org.joda.time.DateTimeZone;
+import org.joda.time.Instant;
```

Comment:
We no longer use jodatime for our datetime-related logic. You need to use the `java.time` equivalents of these objects, e.g. `java.time.Instant`.